### ISSUE TITLE
Setting class modification

### ISSUE THREAD

**Issue by HiIAmTzeKean** [HiIAmTzeKean/Streamsight#28]
The splitting of the timestamp in training set does not raise warning to user if the train or test set is empty.

how this might happen:
user splits at t=5
data starts at t=10

then the train set will be empty since data starts at t=10. We should not throw an exception since this is a user specification, but we will raise a warning to the user
